feature-libs/product/bulk-pricing/core/services/bulk-prices.service.spec.ts

```diff
@@ -1,16 +1,139 @@
 import { TestBed } from '@angular/core/testing';
+import { ProductService } from '@spartacus/core';
+import { of } from 'rxjs/internal/observable/of';
 
 import { BulkPricesService } from './bulk-prices.service';
+const mockProductCode = '2221933';
+
+const mockBulkPrices = {
+  price: {
+     currencyIso: 'USD',
+     formattedValue: '$4.00',
+     priceType: 'BUY',
+     value: 4.0
+  },
+  volumePrices: [ {
+     currencyIso: 'USD',
+     formattedValue: '$4.00',
+     maxQuantity: 9,
+     minQuantity: 1,
+     priceType: 'BUY',
+     value: 4.0
+  }, {
+     currencyIso: 'USD',
+     formattedValue: '$3.89',
+     maxQuantity: 29,
+     minQuantity: 10,
+     priceType: 'BUY',
+     value: 3.89
+  }, {
+     currencyIso: 'USD',
+     formattedValue: '$3.69',
+     maxQuantity: 49,
+     minQuantity: 30,
+     priceType: 'BUY',
+     value: 3.69
+  }, {
+     currencyIso: 'USD',
+     formattedValue: '$3.49',
+     maxQuantity: 99,
+     minQuantity: 50,
+     priceType: 'BUY',
+     value: 3.49
+  }, {
+     currencyIso: 'USD',
+     formattedValue: '$2.99',
+     minQuantity: 100,
+     priceType: 'BUY',
+     value: 2.99
+  } ]
+};
+
+class MockProductService {
```

Comment:
Please provide implements Partial<service> part to keep consistency with other unit tests